pkg/util/utils_linux.go

```diff
@@ -52,3 +53,18 @@ func FindDeviceNodes() (map[string]string, error) {
 
 	return nodes, nil
 }
+
+// CheckRootlessUIDRange checks the uid within the rootless container is in the range from /etc/subuid
+func CheckRootlessUIDRange(uid int) error {
+	uids, _, err := rootless.GetConfiguredMappings()
+	if err != nil {
+		return err
+	}
+	for _, u := range uids {
+		// add 1 since we also map in the user's own UID
+		if uid+1 > u.Size {
```

Comment:
Sorry, not what I meant - should be `uid > u.Size + 1`

Reply:
fixed